ITR/data/base_providers.py

```diff
@@ -465,11 +521,14 @@ def _historic_ei_to_dicts(self, id: str, intensities_scopes: IHistoricEIScopes)
                 intsties = {intsty['year']: intsty['value'] for intsty in intensities}
                 data.append(
                     {ColumnsConfig.COMPANY_ID: id, ColumnsConfig.VARIABLE: VariablesConfig.EMISSIONS_INTENSITIES,
-                     ColumnsConfig.SCOPE: scope, **intsties})
+                     ColumnsConfig.SCOPE: EScope[scope], **intsties})
         return data
 
-    def _compute_missing_historic_ei(self, companies, historic_data):
-        scopes = EScope.get_scopes()
+    # Each benchmark defines its own scope requirements on a per-sector/per-region basis.
+    # The purpose of this function is not to infer scope data that might be interesting,
+    # but rather to impute the scope data that is actually required, no more, no less.
+    def _compute_missing_historic_ei(self, companies: List[ICompanyData], historic_data: pd.DataFrame):
+        scopes = [scope for scope in EScope.get_result_scopes()]
```

Comment:
in original code, this line was `scopes = EScope.get_scopes()`
as i understand, it returned a list of scopes.
why do we need now a more complicated `scopes = [scope for scope in EScope.get_result_scopes()]`?
could it remain a simplier `scopes = EScope.get_result_scopes()`?

Reply:
Changes pushed.